Autopark/Autopark.DataAccess/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,11 @@
+namespace Autopark.DataAccess.Interfaces;
+
+public interface IRepository<T>
+{
+	Task<IEnumerable<T>> GetAllAsync();
```

Comment:
create generic repository class which will implement interface methods, and other repositories will inherit this class, such way you wont have duplicated methods in every repository